app/assets/stylesheets/govuk_publishing_components/components/_action-link.scss

```diff
@@ -73,11 +73,12 @@
 }
 
 .gem-c-action-link__subtext {
-  @include govuk-font(19);
   display: block;
   font-weight: normal;
```

Comment:
Pretty sure that this `font-weight` is redundant and can be removed - it's set by the `govuk-font` mixin.

Reply:
Yup, checked and it is. Thanks